karaf/features/src/main/feature/feature.xml

```diff
@@ -55,7 +55,12 @@
         <bundle>mvn:javax.annotation/javax.annotation-api/${cxf.javax.annotation-api.version}</bundle>
     </feature>
 
+    <feature name="brooklyn-config" version="${project.version}">
```

Comment:
Is it worth declaring the feature `brooklyn-config`, rather than just using `brooklyn-server-config` directly? Did you include it for backwards compatibility of downstream projects, or for another reason?